markdown/blog/2025-march-summary.md

```diff
@@ -0,0 +1,101 @@
+We have opened the Call for Speakers for most of the venues. Please prepare to submit your proposals before each deadline:
+
+- **[AsyncAPI Conference in Munich](https://conference.asyncapi.com/venue/Munich):** Submit your proposal **before May 11**, 2025.
+- **[AsyncAPI Conference in Lagos](https://conference.asyncapi.com/venue/Lagos):** Submit your proposal **before May 17**, 2025.
+- **[AsyncAPI Conference in Bangalore](https://conference.asyncapi.com/venue/Bangalore):** Submit your proposal **before June 29**, 2025.
+- **[AsyncAPI Conference in London](https://conference.asyncapi.com/venue/London):** Submit your proposal **before July 13**, 2025.
+- **[AsyncAPI Conference in Paris](https://conference.asyncapi.com/venue/Paris):** Submit your proposal **before October 5**, 2025.
+
+Please make sure to select the "**AsyncAPI**" option for the track to ensure your submission goes to the correct category.
+
+> **Note:** If your talk does not make it to the in-person events, you can secure a guaranteed spot at the [AsyncAPI online conference](https://conference.asyncapi.com/venue/Online). Don't miss this opportunity to share your insights with the wider community!
+
+
+### AsyncAPI Meetup Section
+We plan to introduce a meetups section under the AsyncAPI Conference umbrella, a platform enabling community members to connect, share knowledge, and discuss topics related to AsyncAPI outside of the main conference events. The goal is to foster collaboration and knowledge-sharing within the AsyncAPI ecosystem.
+
+As part of preparing the section, we will have an open discussion on the guidelines and rules. Additionally, we have an open issue with the design for implementing the page for the conference website. I encourage everyone to share their thoughts and [review the AsyncAPI Meetups Section discussion](https://github.com/asyncapi/conference-website/discussions/604) and [the AsyncAPI Meetups Page issue](https://github.com/asyncapi/conference-website/issues/644).
+
+## Spec x Tooling
+
+We are excited to welcome a new project, [Neuroglia.AsyncAPI](https://github.com/asyncapi/net-sdk), a .NET SDK for visualizing and interacting with AsyncAPI documents donated by [Charles d'Avernas](https://www.linkedin.com/in/charles-d-avernas-40836a13/). The project also supports a code-first documentation generator, fluent builders for intuitive and expressive API definitions, and fluent validators to ensure compliance and correctness. All the components also support the AsyncAPI v3. [Be sure to try out Neuroglia.AsyncAPI for your documents](https://github.com/asyncapi/net-sdk), especially if you are a .NET developer!
+
+### RAML Schema Parser
+
+We are looking to archive the RAML schema parser, a standard for RAML data types, unless a community member is interested in maintaining the project. If you have any thoughts on this issue, [please share them in the open discussion about the RAML Parser](https://github.com/asyncapi/raml-dt-schema-parser/issues/255).
```

Comment:
```suggestion
At the moment we do not have active maintianers in RAML schema parser, a standard for RAML data types. If there is any community member who depends on this parser and is interested in maintaining the project, reach out to us. If you have any thoughts on this issue, [please share them in the open discussion about the RAML Parser](https://github.com/asyncapi/raml-dt-schema-parser/issues/255).
```

just suggesting different messaging, but probably can be written better than I did